src/components/App.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import ButtonPanel from './ButtonPanel';
+import Display from './Display';
+
+const App = () => (
+  <div>
+    <header>
```

Comment:
The HTML `<header>` element represents introductory content, typically a group of introductory or navigational aids. It's not a good use case here because it's wrapping all your app.